tableschema/types/array.py

```diff
@@ -11,7 +11,7 @@
 
 # Module API
 
-def cast_array_default(value):
+def cast_array(format, value):
```

Comment:
Not sure if there is a previous context on this but wouldn't it make more sense to move `format` to an optional keyword argument that defaults to `default` as this is what people will need 90% of the time: `cast_*(value)`. Others can use `cast_string(value, format='email')` or just `cast_string(value, 'email')`.